src/components/Edit/ChangeSummary.vue

```diff
@@ -0,0 +1,155 @@
+        <v-col cols="9">
+          <v-icon color="primary">mdi-file-document-edit-outline</v-icon>
+          <label class="summary-title">Summary of Changes to File</label>
+        </v-col>
+
+        <!-- Actions -->
+        <v-col cols="3" class="mt-n2">
+          <div class="actions mr-4">
+            <v-btn
+              text color="primary"
+              id="btn-change"
+              :disabled="isBusySaving"
+              @click="onChangeClicked()"
+            >
+              <v-icon small>mdi-pencil</v-icon>
+              <span>Change</span>
+            </v-btn>
+            <v-btn
+              text color="primary"
+              id="btn-delete"
+              :disabled="isBusySaving"
+              @click="onDeleteClicked()"
+            >
+              <v-icon small>mdi-delete</v-icon>
+              <span>Delete</span>
+            </v-btn>
+          </div>
+        </v-col>
+      </v-row>
+    </div>
+
+    <!-- Business Name -->
+    <template v-if="hasBusinessNameChanged">
+      <v-divider class="mx-4" />
+      <div class="section-container business-name-summary">
+        <v-row no-gutters>
+          <v-col cols="3">
+            <label><strong>Company Name</strong></label>
+          </v-col>
+
+          <v-col cols="8" class="mt-n1">
+            <div class="company-name font-weight-bold text-uppercase">{{ companyName }}</div>
+            <div class="company-name mt-2">{{ getNameRequest.nrNumber }}</div>
+          </v-col>
+        </v-row>
+      </div>
+    </template>
+  </v-card>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Mixins, Prop } from 'vue-property-decorator'
+import { Action, Getter } from 'vuex-class'
+import { ActionBindingIF } from '@/interfaces'
+import { DateMixin, EnumMixin, FilingTemplateMixin, LegalApiMixin, PayApiMixin } from '@/mixins'
+
+@Component({
+  components: {}
+})
+export default class ChangeSummary extends Mixins(
+  DateMixin,
+  EnumMixin,
+  FilingTemplateMixin,
+  LegalApiMixin,
+  PayApiMixin
+) {
+  // Global getters
+  @Getter getApprovedName!: string
+  @Getter getBusinessNumber!: string
+  @Getter isBusySaving!: boolean
+
+  // Change flag getters
+  @Getter hasBusinessNameChanged!: boolean
+
+  // Global actions
+  @Action setSummaryMode!: ActionBindingIF
+
+  /** Prop to perform validation. */
+  @Prop() readonly validate: boolean
+
+  /** The company name (from NR, or incorporation number). */
+  get companyName (): string {
+    if (this.getApprovedName) return this.getApprovedName
+
+    return `${this.getBusinessNumber || '[Incorporation Number]'} B.C. Ltd.`
+  }
+
+  async onChangeClicked (): Promise<void> {
+    this.setSummaryMode(false)
+    // We don't change views just interchange components, so scroll to top for better UX.
+    await this.scrollToTop(document.getElementById('app'))
+  }
+
+  onDeleteClicked (): void {
+    this.$root.$emit('delete-all')
+  }
+
+  @Emit('haveChanges')
+  emitHaveChanges (): void {}
```

Comment:
Not used?

Reply:
Updated! 

Was a left over from the template i used for this placeholder component. 
I'm sure it will be used in the near future, but definitely not used at the moment.